sandbox/correct-reads.py

```diff
@@ -330,43 +333,46 @@ def main():
                     written_reads += 1
                     written_bp += len(new_seq)
 
-        print >>sys.stderr, 'removing %s' % pass2filename
+        print('removing %s' % pass2filename, file=sys.stderr)
         os.unlink(pass2filename)
 
-    print >>sys.stderr, 'removing temp directory & contents (%s)' % tempdir
+    print('removing temp directory & contents (%s)' % tempdir, file=sys.stderr)
     shutil.rmtree(tempdir)
 
     n_passes = 1.0 + (float(save_pass2_total) / n_reads)
     percent_reads_corrected = float(corrected_reads +
                                     (n_reads - written_reads)) /\
         n_reads * 100.0
 
-    print >>sys.stderr, 'read %d reads, %d bp' % (n_reads, n_bp,)
-    print >>sys.stderr, 'wrote %d reads, %d bp' % (written_reads, written_bp,)
-    print >>sys.stderr, 'looked at %d reads twice (%.2f passes)' % \
-        (save_pass2_total, n_passes)
-    print >>sys.stderr, 'removed %d reads and corrected %d reads (%.2f%%)' % \
-        (n_reads - written_reads, corrected_reads, percent_reads_corrected)
-    print >>sys.stderr, 'removed %.2f%% of bases (%d total)' % \
-        ((1 - (written_bp / float(n_bp))) * 100.0, n_bp - written_bp)
+    print('read %d reads, %d bp' % (n_reads, n_bp,), file=sys.stderr)
+    print('wrote %d reads, %d bp' % (written_reads, written_bp,),
+          file=sys.stderr)
+    print('looked at %d reads twice (%.2f passes)' %
+          (save_pass2_total, n_passes), file=sys.stderr)
+    print('removed %d reads and corrected %d reads (%.2f%%)' %
+          (n_reads - written_reads, corrected_reads, percent_reads_corrected),
+          file=sys.stderr)
+    print('removed %.2f%% of bases (%d total)' %
+          ((1 - (written_bp / float(n_bp))) * 100.0, n_bp - written_bp),
+          file=sys.stderr)
```

Comment:
Bless you for the tedious and thankless work towards Python 2/3 compatibility! 👍